tests/router_test.js

```diff
@@ -4595,6 +4596,84 @@ test("A failed handler's setup shouldn't prevent future transitions", function(a
     assert.equal(barModelCount, 1, 'Bar model should be called once');
   });
 
+  test('Calling replaceWith after initial replace in validation hook with multiple redirects should use replaceUrl', function(assert) {
+    // assert.expect(8);
```

Comment:
Why is this commented?

Reply:
I commented it out since I kept changing the number of asserts. Should probably just remove it since I'm not doing any asserts in callbacks.

Reply:
Ya, makes sense